### ISSUE TITLE
v3AAC Tower Bypass

### ISSUE THREAD

**Issue by Rqcco** [konsolas/AAC-Issues#149]
Towering blocks works https://www.youtube.com/watch?v=su6wDdgFE0E

**Comment by Janmm14** [comment]
I suggest especially blocking tower hacks, as the rate legit clients can build themselves up is pretty easy:
2 blocks/sec

**Comment by Photon-GitHub** [comment]
The problem is the calculation:
Tower uses the mechanic of step, which is pretty hard to detect, and lag is another false-positive factor. Tower can for sure be patched to a certain degree, the rest is the problem of detecting Step.

**Comment by konsolas** [comment]
I'm going to close this as wontfix since the tower hack in question seems to be more automated than gaining an advantage. Most faster towers are blocked by the fly check.
